packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleAnchorButtonTests.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Linq.Expressions;
+using Bunit;
+using Xunit;
+
+namespace NimbleBlazor.Tests.Unit.Components;
+
+/// <summary>
+/// Tests for <see cref="NimbleAnchorButton"/>.
+/// </summary>
+public class NimbleAnchorButtonTests
+{
+    [Fact]
+    public void NimbleAnchorButton_Render_HasAnchorButtonMarkup()
+    {
+        var context = new TestContext();
+        context.JSInterop.Mode = JSRuntimeMode.Loose;
+        var expectedMarkup = "nimble-anchor-button";
+
+        var button = context.RenderComponent<NimbleAnchorButton>();
+
+        Assert.Contains(expectedMarkup, button.Markup);
+    }
+
+    [Theory]
+    [InlineData(ButtonAppearance.Block, "block")]
+    [InlineData(ButtonAppearance.Outline, "outline")]
+    [InlineData(ButtonAppearance.Ghost, "ghost")]
+    public void ButtonAppearance_AttributeIsSet(ButtonAppearance value, string expectedAttribute)
+    {
+        var button = RenderWithPropertySet(x => x.Appearance, value);
+
+        Assert.Contains(expectedAttribute, button.Markup);
+    }
+
+    [Theory]
+    [InlineData(ButtonAppearanceVariant.Default, "<nimble-anchor-button>")]
+    [InlineData(ButtonAppearanceVariant.Primary, "appearance-variant=\"primary\"")]
+    public void ButtonAppearanceVariant_AttributeIsSet(ButtonAppearanceVariant value, string expectedAttribute)
```

Comment:
Maybe rename expectedAttribute to expectedMarkup as the first string contains no attributes?

Reply:
Done